src/WinRT.Runtime2/Properties/WindowsRuntimeFeatureSwitches.cs

```diff
@@ -51,6 +56,11 @@ internal static class WindowsRuntimeFeatureSwitches
     [FeatureSwitchDefinition(UseWindowsUIXamlProjectionsPropertyName)]
     public static bool UseWindowsUIXamlProjections { get; } = GetConfigurationValue(UseWindowsUIXamlProjectionsPropertyName, defaultValue: false);
 
+    /// <summary>
+    /// Gets a value indicating whether or not should initialize all default type mappings automatically (defaults to <see langword="true"/>).
+    /// </summary>
+    public static bool EnableDefaultCustomTypeMappings { get; } = GetConfigurationValue(EnableDefaultCustomTypeMappingsPropertyName, defaultValue: false);
```

Comment:
Why are we adding this switch? It shouldn't be applicable anymore in CsWinRT 3.0 (no static initializers)